build/root/Makefile.generated_files

```diff
@@ -469,6 +472,17 @@ $(OPENAPI_OUTFILE): $(OPENAPI_GEN) $(KNOWN_VIOLATION_FILENAME)
 	diff $(REPORT_FILENAME) $(KNOWN_VIOLATION_FILENAME) ||                          \
 	(echo -e $(API_RULE_CHECK_FAILURE_MESSAGE); exit 1)
 
+# TODO(roycaihw): write Makefile in apiextensions-apiserver
+$(CRD_OPENAPI_OUTFILE): $(OPENAPI_GEN)
+	./hack/run-in-gopath.sh $(OPENAPI_GEN)                                          \
+	    --v $(KUBE_VERBOSE)                                                         \
+	    --logtostderr                                                               \
+	    -i "k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/api/autoscaling/v1"         \
```

Comment:
metav1.TypeMeta, metav1.ObjectMeta and autoscalingv1.Scale are referred by CR schema. we need to pre-generate the definitions to publish them from apiextensions-apiserver

also the definitions must deep-equal to the ones we served for built-in APIs, otherwise kube-openapi dedups the definitions by renaming